kube/base/deployment.yaml

```diff
@@ -20,6 +20,9 @@ spec:
       containers:
       - name: heimdall
         image: quay.io/uswitch/heimdall
+        envFrom:
+        - secretRef:
+            name: heimdall-sentry-secret
```

Comment:
This should probably be in our overlay in our own repo as this would force adopters to have a sentry secret if they want to use the manifests from this repo.

Reply:
https://github.com/uswitch/k8s-heimdall/pull/8

Reply:
This envFrom section will be removed with the next update :+1: 
The other PR mentioned above adds this section instead